src/encoded/tests/test_audit_experiment.py

```diff
@@ -3298,3 +3298,39 @@ def test_audit_experiment_histone_characterized_no_primary(testapp,
     res = testapp.get(base_experiment['@id'] + '@@index-data')
     assert any(error['category'] == 'antibody not characterized to standard'
                for error in collect_audit_errors(res))
+
+
+def test_audit_experiment_tag_target(testapp, experiment, ctcf):
+    tag_target = testapp.post_json(
+        '/target',
+        {'genes': [ctcf['uuid']],
```

Comment:
Another example, the outer parens are dropped to the next line so we should follow this pattern in the code block...
```
    tag_target = testapp.post_json(
        '/target',
        {
            'genes': [ctcf['uuid']],
            'modifications': [{'modification': 'eGFP'}],
            'label': 'eGFP-CTCF',
            'investigated_as': ['recombinant protein'],
        }
    ).json['@graph'][0]
```